pybikes/bixi.py

```diff
@@ -168,22 +168,22 @@ def from_json(data):
     @staticmethod
     def from_json_xml(data):
         """ json marker object translated from xml
-        { 
-            "id": "2", 
-            "name": "Docklands Drive - Docklands", 
-            "terminalName": "60000", 
-            "lastCommWithServer": "1375644471147", 
-            "lat": "-37.814022", 
-            "long": "144.939521", 
-            "installed": "true", 
-            "locked": "false", 
-            "installDate": "1313724600000", 
-            "removalDate": {  }, 
-            "temporary": "false", 
-            "public": "true", 
-            "nbBikes": "15", 
-            "nbEmptyDocks": "8", 
-            "latestUpdateTime": "1375592453128" 
+        {
+            "id": "2",
+            "name": "Docklands Drive - Docklands",
+            "terminalName": "60000",
+            "lastCommWithServer": "1375644471147",
+            "lat": "-37.814022",
+            "long": "144.939521",
+            "installed": "true",
+            "locked": "false",
+            "installDate": "1313724600000",
+            "removalDate": {  },
+            "temporary": "false",
+            "public": "true",
+            "nbBikes": "15",
+            "nbEmptyDocks": "8",
+            "latestUpdateTime": "1375592453128"
         }
```

Comment:
This part of the commit is not necessary, unless you do not mind appeariong on the `git blame` for it